src/earn/EarnConfirmationScreen.tsx

```diff
@@ -240,50 +257,24 @@ function CollectItem({
             tokenId={tokenInfo.tokenId}
             amount={rewardAmount}
             showLocalAmount={false}
-            testID={`EarnCollect/${tokenInfo.tokenId}/CryptoAmount`}
+            testID={`EarnConfirmation/${tokenInfo.tokenId}/CryptoAmount`}
           />
           <TokenDisplay
             style={styles.fiatText}
             tokenId={tokenInfo.tokenId}
             amount={rewardAmount}
             showLocalAmount={true}
-            testID={`EarnCollect/${tokenInfo.tokenId}/FiatAmount`}
+            testID={`EarnConfirmation/${tokenInfo.tokenId}/FiatAmount`}
           />
         </View>
       </View>
     </>
   )
 }
 
-function Rate({ pool }: { pool: EarnPosition }) {
```

Comment:
I checked with design and this rate is expected to be removed as it was confusing users.